src/tests/daos_perf.c

```diff
@@ -548,6 +548,7 @@ akey_update_or_fetch(int obj_idx, enum ts_op_type op_type,
 		fprintf(stderr, "%s failed. rc=%d, epoch=%"PRIu64"\n",
 			op_type == TS_DO_FETCH ? "Fetch" : "Update",
 			rc, *epoch);
+		dts_credit_return(&ts_ctx, cred);
```

Comment:
Does this need to be protected by param->pa_rw.verify like line 559 ?

Reply:
fixed, thx